src/fides/config/__init__.py

```diff
@@ -69,9 +70,7 @@ class FidesConfig(FidesSettings):
     admin_ui: AdminUISettings
     consent: ConsentSettings
     cli: CLISettings
-    celery: Dict = Field(
-        description="This section can be used to pass config vars to Celery directly.",
-    )
+    celery: CelerySettings
```

Comment:
Other vals set in the toml file can still come in through here I believe because CelerySettings > FidesSettings has extra > allow_all.  These will get passed through to the celery_app.conf

Reply:
thanks for clarifying - i was actually just wondering how those got passed through!

based on your testing notes, it seems like you've tested a "mix" of both .toml file "extra" attributes alongside env vars for our "expected" attributes. is that correct? i'd just want to make sure that's covered (seems like you've covered it). i can also test on my end to confirm :)

Reply:
I did test a mix, i can see how my instructions didn't spell that out though!

Reply:
heh no your instructions were pretty clear :) i just wanted to be extra sure 👍